logstash-core/logstash-core.gemspec

```diff
@@ -47,17 +47,17 @@ Gem::Specification.new do |gem|
 
   gem.platform = "java"
 
-  gem.add_runtime_dependency "pry", "~> 0.10.1"  #(Ruby license)
+  gem.add_runtime_dependency "pry", "~> 0.12.2"  #(Ruby license)
   gem.add_runtime_dependency "stud", "~> 0.0.19" #(Apache 2.0 license)
   gem.add_runtime_dependency "clamp", "~> 0.6.5" #(MIT license) for command line args/flags
-  gem.add_runtime_dependency "filesize", "0.0.4" #(MIT license) for :bytes config validator
-  gem.add_runtime_dependency "gems", "~> 0.8.3"  #(MIT license)
-  gem.add_runtime_dependency "concurrent-ruby", "~> 1.0", ">= 1.0.5"
+  gem.add_runtime_dependency "filesize", "~> 0.2.0" #(MIT license) for :bytes config validator
+  gem.add_runtime_dependency "gems", "~> 1"  #(MIT license)
+  gem.add_runtime_dependency "concurrent-ruby", "~> 1"
   gem.add_runtime_dependency "rack", '~> 1.6', '>= 1.6.11'
-  gem.add_runtime_dependency "sinatra", '~> 1.4', '>= 1.4.6'
-  gem.add_runtime_dependency 'puma', '~> 2.16'
-  gem.add_runtime_dependency "jruby-openssl", ">= 0.9.20" # >= 0.9.13 Required to support TLSv1.2
-  gem.add_runtime_dependency "chronic_duration", "0.10.6"
+  gem.add_runtime_dependency "sinatra", '~> 1', '>= 1.4.6'
+  gem.add_runtime_dependency 'puma', '~> 2'
```

Comment:
I'm also wary of the `~> 2` here; if puma ever releases a 3.x, we'll silently upgrade which could be hard to debug.